--- a/setup.py
+++ b/setup.py
@@ -55,8 +55,6 @@
         if distro in ['centos', 'redhat', 'debian']:
             data_files.append(('/etc/init.d',
                                ['bin/init.d/diamond']))
-            data_files.append(('/var/log/diamond',
-                               ['.keep']))
             if distro_major_version >= '6' and not distro == 'debian':
                 data_files.append(('/etc/init',
                                    ['rpm/upstart/diamond.conf']))
